src/components/pages/ParentContact/ParentContactContainer.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
+import { Layout, Row, Col } from 'antd';
+import img2 from './img2.png';
+import ParentNavTopBar from '../../common/ParentNavTopBar';
+
+const ParentContactContainer = () => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [message, setMessage] = useState('');
+  const [item, setItem] = useState([]);
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    setItem([...item, { name: name, email: email, message: message }]);
+    setName('');
+    setEmail('');
+    setMessage('');
+  };
+
+  useEffect(() => {
+    const data = localStorage.getItem('data');
+    if (data) {
+      setItem(JSON.parse(data));
+    }
+  }, []);
```

Comment:
Curious to know what is going on in this area of the code - it might be good to comment this section for incoming teams also.

Reply:
It is for storing the info in the local storage. Because we have no specific info whether the data will goes to any database or use a third-party software to store the info.